make/Makefile.test

```diff
@@ -15,6 +15,67 @@ ifeq ("$(VAGRANT)","")
 VAGRANT="/usr/bin/vagrant"
 endif
 
+_PACKER_ARGS?=
```

Comment:
that's a public one, pipelines are using it and it allows to enable/disable features during install